docs/reference/examples/exploitation.json

```diff
@@ -0,0 +1,24 @@
+{
+  "namespace": "ssvc",
+  "version": "1.0.0",
+  "name": "Exploitation",
+  "description": "The present state of exploitation of the vulnerability.",
+  "key": "E",
+  "values": [
+    {
+      "name": "None",
+      "description": "There is no evidence of active exploitation and no public proof of concept (PoC) of how to exploit the vulnerability.",
+      "key": "N"
+    },
+    {
+      "name": "PoC",
+      "description": "One of the following cases is true: (1) private evidence of exploitation is attested but not shared; (2) widespread hearsay attests to exploitation; (3) typical public PoC in places such as Metasploit or ExploitDB; or (4) the vulnerability has a well-known method of exploitation.",
```

Comment:
Do we actually think (1) and (2) are operationalizable? 
CISA's SSVC version of *State of Exploitation* removes them https://www.cisa.gov/sites/default/files/publications/cisa-ssvc-guide%20508c.pdf
Maybe we should move for consistency between instances of the decision point?

Reply:
Separate issue.

Reply:
#353